### PR TITLE
feat: Drop testing on node4

### PR DESCRIPTION
**Pull request by wagnerand**
See https://github.com/mozilla/addons-linter/pull/1272

Fixes https://github.com/mozilla/web-ext/issues/887

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ sudo: false
 
 # Keep this in sync with appveyor.yml
 node_js:
-- '4'
 - '6'
```
Comment (kumar303) [inline .travis.yml]: Can you add Node 8 to the list? That will help us prepare for the next [LTS release](https://github.com/nodejs/LTS). If adding that causes problems, we can do it in a separate issue but I don't foresee any issues.

**Comment by kumar303** [comment]
Once you push a new commit (don't rebase or squash) then the build error will be resolved.

File: .travis.yml (hunk 1 above)
Comment (wagnerand) [inline .travis.yml]: But...there is no Node 8 yet?

Comment (kumar303) [inline .travis.yml]: Hah, sorry, I thought there was. Nevermind then!

**Comment by kumar303** [comment]
Don't sweat the changelog linter, it's being annoying right now. When I squash and merge I'll make sure the final commit is formatted correctly.

**Pull request merged**